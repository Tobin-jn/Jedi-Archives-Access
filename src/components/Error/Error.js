import React from 'react';

const Error = () => {
  return(
    <div className="error">
      <h3>Error</h3>
    </div>
  )
}

export default Error