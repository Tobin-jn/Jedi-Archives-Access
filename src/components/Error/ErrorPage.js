import React from 'react';
import PropTypes from 'prop-types';
import './ErrorPage.css';

const ErrorPage = ({ errorState }) => {
  return(
    <div 
      className={`error-page ${errorState}`}
      aria-label='error-page'
    >
      <h1 className="error">Connection lost...</h1>
    </div>
  );
}

ErrorPage.propTypes = {
  errorState: PropTypes.string.isRequired,
};

export default ErrorPage;