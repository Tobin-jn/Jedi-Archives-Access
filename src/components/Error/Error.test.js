import React from 'react';
import Error from './ErrorPage';
import { shallow } from 'enzyme';

let wrapper;


beforeEach(() => {
  wrapper = shallow(<Error errorState={true}

  />)
});

describe('Error', () => {
  it ('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});