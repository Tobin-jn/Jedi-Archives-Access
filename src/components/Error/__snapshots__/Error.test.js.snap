// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
<div
  className="error-page undefined"
>
  <h1
    className="error"
  >
    Connection lost...
  </h1>
</div>
`;
